pkg/kudoctl/cmd/client/client.go

```diff
@@ -0,0 +1,35 @@
+package client
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/clog"
+	kudoinit "github.com/kudobuilder/kudo/pkg/kudoctl/cmd/init"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/env"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/kube"
+	"github.com/kudobuilder/kudo/pkg/kudoctl/util/kudo"
+)
+
+// GetClient is a helper function that takes the Settings struct and returns a new KUDO Client
+func GetValidatedClient(s *env.Settings) (*kudo.Client, error) {
```

Comment:
NewValidatedClient?

Reply:
We should NOT have `client.GetValidatedClient` or `client.NewValidatedClient` or `client.NewClient` or `client.Client` :) 

This does not communicate anything to the developer using or reading the code.

Reply:
Agree with Ken, package name is Go is quite useful und repeating it in the method name is considered bad practice (along with [`Get`ers and `Set`ers](https://golang.org/doc/effective_go.html#Getters): wdyt about `client.New` and `client.Validated`?